app/DTO/AlbumDTO.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\DTO;
+
+use App\Contracts\AbstractAlbum;
+use App\Models\Album;
+
+/**
+ * Data Transfer Object (DTO) for Albums.
+ *
+ * This allows us to decorate the Album with its associated current user rights.
+ */
+class AlbumDTO extends DTO
```

Comment:
```suggestion
class AbstractAlbumDTO extends DTO
```

The class transfers an `AbstractAlbum`, then it should also be called like it.